sdk/textanalytics/Azure.AI.TextAnalytics/src/JobManifestTasks.cs

```diff
@@ -11,21 +11,21 @@ namespace Azure.AI.TextAnalytics
     /// JobManifestTasks.
     /// </summary>
     [CodeGenModel("JobManifestTasks")]
-    public partial class JobManifestTasks
```

Comment:
is there a reason why we can't just delete the file?
it is ok if you don't want to do it now, but please create an issue so this can be done for the next sprint.

Reply:
Applies to all other internal types like I mentioned in above comment

Reply:
autogenerator did not create internal files so i was not able to delete these.
created an issue https://github.com/Azure/azure-sdk-for-net/issues/18507